ts/components/session/LeftPaneMessageSection.tsx

```diff
@@ -291,4 +396,61 @@ export class LeftPaneMessageSection extends React.Component<Props, any> {
       });
     }
   }
+
+  private handleJoinPublicChat() {
```

Comment:
We shouldn't be making an almost exact copy of `handleJoinChannelButtonClick`, needs refactoring.

Reply:
Done.